www/template/tutor-list-directive.html

```diff
@@ -0,0 +1,5 @@
+<!-- tutor-list-directive -->
+
+<ion-list>
```

Comment:
the goal was in this step to add a non-styled list, but it's alright, mark the appropriate todos checked then